src/wp-includes/deprecated.php

```diff
@@ -3349,6 +3351,8 @@ function gd_edit_image_support($mime_type) {
 				return function_exists('imagecreatefrompng');
 			case 'image/gif':
 				return function_exists('imagecreatefromgif');
+			case 'image/webp':
+				return function_exists('imagecreatefromwebp');
```

Comment:
This needs to be wrapped in a `function_exists` check like elsewhere in this PR.